test/evergreen.yml

```diff
@@ -768,9 +768,9 @@ functions:
     - command: archive.targz_pack
       params:
         target: ${upload_filename|wiredtiger.tgz}
-        source_dir: ${upload_source_dir|wiredtiger}
+        source_dir: ${upload_parent_dir|.}
         include:
-          - "./**"
+          - "${upload_include|wiredtiger/**}"
```

Comment:
Evergreen's `targz_pack` function seems to work by finding all files that match the provided regex, and when we pass in `./**` with a `cwd` of wiredtiger we lose that wiredtiger is the top level folder. This change captures the wiredtiger folder and adds it to the tarball.